### PR TITLE
add Rhee

### PR DESCRIPTION
**Pull request by shivapbhusal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ PHP
 
 Python
 ---------
+- [Rhee - a general purpose Nepali programming language](https://github.com/biprasu/mrhee)
```
Comment (geshan) [inline README.md:33]: Please follow the alphabetical ordering.

**Review by geshan — CHANGES_REQUESTED** [review]
needs changes.

File: README.md (hunk 1 above)
Comment (shivapbhusal) [reply README.md:33]: Thanks for your comment. Please review the new pull request. Thanks.

**Review by techgaun — CHANGES_REQUESTED** [review]
looks like dup of other.

**Pull request closed without merging**